lib/json_api/request/builder.rb

```diff
@@ -36,8 +34,8 @@ def put(payload)
       end
 
       def method_missing(method, *args, **kwargs, &block)
```

Comment:
We definitely should. I'd want to add some tests for that, though, which puts it outside the scope of this PR.